s3/src/test/scala/akka/stream/alpakka/s3/scaladsl/S3IntegrationSpec.scala

```diff
@@ -45,7 +47,7 @@ trait S3IntegrationSpec
   )
   implicit val ec: ExecutionContext = actorSystem.dispatcher
 
-  implicit val defaultPatience: PatienceConfig = PatienceConfig(90.seconds, 100.millis)
+  implicit val defaultPatience: PatienceConfig = PatienceConfig(3.minutes, 100.millis)
```

Comment:
I had to increase this to 3 minutes because depending on laptop spec/internet connection it can be possible that you cannot download/upload the large chunks for the related tests in time.